source/includes/metype/_user_generated_content.md

```diff
@@ -50,3 +50,8 @@ Please find the script in the shell on the right in the shell.
 
 * Setup the embed code for the UGC widget at the position you need to embed.
 Please find the script with detailed comments on the right in the shell.
+
+* You can pass the extra story attribute in UGC. All you want to set is data attributes `data-metype-topic-type` and `data-metype-topic-value` in the div with `contribution-container` id. You can set this data attributes from front-end with the help of javascript.
```

Comment:
`You can pass the extra story attribute in UGC` -> `You can also pass the topic for UGC.`

Reply:
@himani39 In the example, we are setting story-attributes.name_of_attribute. So i think, it would be better if we mention it as story attribute instead of topic. What about this -> `You can also pass the attribute of story in UGC.`

Reply:
ok, then I think we can mention it as `You can also pass the topic as attribute of story in UGC`.

 Also, just noticed that example also needs modification. It should be `data-metype-topic-type="story-attributes.ugctopic" instead of `data-metype-topic-type="story-attributes.sports"`. This is beacuse sports itself is a topic.

Reply:
@himani39 Done.